sct.py

```diff
@@ -108,6 +108,13 @@
 
 LOGGER = setup_stdout_logger()
 
+# #################### constants for hydra list-images output ####################
+VERSION_FIELDS = ["Backend", "Name", "ImageId", "CreationDate"]
+VERSION_FIELDS_WITH_TAG_NAME = VERSION_FIELDS + ["NameTag"]
+#  TODO: align branch and version fields once scylla-pkg#2995 is resolved
+BRANCH_SPECIFIC_FIELDS = ["BuildId", "Arch", "ScyllaVersion"]
+BRANCH_FIELDS = VERSION_FIELDS + BRANCH_SPECIFIC_FIELDS
+BRANCH_FIELDS_WITH_TAG_NAME = VERSION_FIELDS_WITH_TAG_NAME + BRANCH_SPECIFIC_FIELDS
```

Comment:
This is used in one specific function.
Why it was decided to place it here?
I really fail to see any gain in such a move.